client/options/specs.ts

```diff
@@ -103,20 +103,22 @@ export const specs: { [id: string]: OptionSpec } = {
 		noExecOnStart: true,
 		exec: initNowPlaying,
 	},
-	// Random video player
-	meguTV: {
+	// User-specified video in the background
+	bgVideo: {
+		type: optionType.menu,
+		default: "none",
 		noExecOnStart: true,
-		exec: initTV,
+		exec: renderBackground,
 	},
-	// Illya dance in the background
-	illyaDance: {
+	// Mute user-specified background video
+	bgMute: {
 		noExecOnStart: true,
 		exec: renderBackground,
 	},
-	// Mute Illya dance
-	illyaDanceMute: {
+	// Random video player
+	meguTV: {
 		noExecOnStart: true,
-		exec: renderBackground,
+		exec: initTV,
```

Comment:
Why the change?

Reply:
That's not actually changed, it's github being weird with showing the revision changes.
Please see the actual file.